Formula/s/sq.rb

```diff
@@ -0,0 +1,27 @@
+class Sq < Formula
+  desc "Data wrangler with jq-like query language"
+  homepage "https://sq.io"
+  url "https://github.com/neilotoole/sq/archive/refs/tags/v0.48.3.tar.gz"
+  sha256 "46e75e2db83a6cbc98b07dbcfb23de03fc41b2b2cbc7de7aaee0425cef4fb9bb"
+  license "MIT"
+
+  depends_on "go" => :build
+  uses_from_macos "sqlite" => :test
+
+  def install
+    system "go", "build", *std_go_args(ldflags: "-s -w")
```

Comment:
need to match with this, https://github.com/neilotoole/sq/blob/e86d01fdb756a212c59edd3bd6cbd5a5a1b3056d/Makefile#L6

Reply:
Thanks for the catch, @chenrui333! Not sure how I missed seeing the Makefile. 🤦‍♂️